tenants/ai-example/single-model-serving/base/main/create-bucket-job.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    app.kubernetes.io/component: minio
+    app.kubernetes.io/instance: minio
+    app.kubernetes.io/name: minio
+    app.kubernetes.io/part-of: minio
+    component: minio
+  name: create-minio-buckets
+  namespace: ai-example-single-model-serving
+spec:
+  selector: {}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: minio
+        app.kubernetes.io/instance: minio
+        app.kubernetes.io/name: minio
+        app.kubernetes.io/part-of: minio
+        component: minio
+    spec:
+      containers:
+      - name: create-buckets
+        image: image-registry.openshift-image-registry.svc:5000/redhat-ods-applications/s2i-generic-data-science-notebook:2023.1
+        imagePullPolicy: IfNotPresent
+        command:
+          - /bin/bash
+        env:
+          - name: BUCKETNAME
+            value: "single-model-bucket"
+        envFrom:
+          - secretRef:
+              name: single-model-serving-ds-connection
+        args:
+          - -ec
+          - |-
+            oc get secret single-model-serving-ds-connection
+            env | grep AWS_
+            cat << 'EOF' | python3
```

Comment:
Instead of doing the indirection for the python script I would recommend creating it as a normal python file and then you can use a configmap generator in kustomize to turn it into a cm that can be mounted in the job.

See an example here:
https://github.com/redhat-ai-services/ai-accelerator/tree/main/components/operators/gpu-operator-certified/instance/components/aws-gpu-machineset

Reply:
fixed: moved the scripts out of the jobs and using kustomize to create config map